.travis.yml

```diff
@@ -2,5 +2,3 @@ language: node_js
 node_js:
   - '5'
```

Comment:
4 is LTS so we can't drop that one, and 6 will have an LTS version in October. No one will use 5 anymore.